Signal/src/contact/OWSContactsManager.m

```diff
@@ -7,6 +7,8 @@
 
 typedef BOOL (^ContactSearchBlock)(id, NSUInteger, BOOL *);
 
+NSString *const OWSContactsManagerContactsDidChangeNotification = @"OWSContactsManagerContactsDidChangeNotification";
```

Comment:
nit: This is a little misleading, because we fire this in response to updating the `SignalRecipients` (via the poorly named `ContactsUpdater`, not explicitly after the Contacts change. 

Though, through a chain of notifications we update our SignalRecipients anytime the Contacts are updated, I think they're currently equivalent, but `OWSContactsManagerSignalRecipientsDidChangeNotification` or `OWSContactsManagerDidIntersectContactsNotification` might have less surprise potential down the road.

Or maybe I'm thinking too hard about it.